### ISSUE TITLE
Define Industry Standard Mechanism for Verifying SIVIC Execution

### ISSUE THREAD

**Issue by graphicdeveloper** [InteractiveAdvertisingBureau/SIMID#207]
The SIVIC working group call on 2/14/19 discussed the need to define a mechanism by which the execution of a SIVIC Interactive Creative will be tracked and verified. As of yet the group has no proposal. Conversation touched on the notion that this may be appropriate for OMID. It may also be that non-OMID-enabled players require a different solution.

Perhaps a single message from Player to SIVIC Creative that includes an Ad Session ID could serve to correlate execution pixels sent by both the Player and the Creative? Maybe the [TRANSACTIONID] macro value would work, or maybe a new macro / tracking event should be added to VAST 4.1.x? 

Bottom line - it's predictable that marketers will complain about paying a SIVIC impression price if they can't verify that the SIVIC Interactive Creative executed in addition to the video file (otherwise it's just a VAST Media ad, right?). We should put our collective wisdom to use in solving this before the final roll out of SIVIC 1.0.

**Comment by amitshetty** [comment]
We will leave this around until the OM group asks for something in the SIMID API or specs... Unclear we have to do anything right now.
